common/api/core-backend.api.md

```diff
@@ -4588,6 +4597,22 @@ export class Schemas {
     static unregisterSchema(schemaName: string): boolean;
 }
 
+// @beta (undocumented)
```

Comment:
beta APIs require documentation.

Reply:
I see no reason for this to be public. I think it should be @internal.